selenium-system-test/src/test/scala/hts/steps/Steps.scala

```diff
@@ -24,16 +24,17 @@ import hts.driver.Driver
 import org.openqa.selenium.WebDriver
 import org.scalatest.Matchers
 
+
 trait Steps extends ScalaDsl with EN with Matchers {
 
   import Steps._
 
   /** Tries to get the value of [[_driver]] - will throw an exception if it doesn't exist */
-  implicit def getDriverUnsafe: WebDriver = _driver.getOrElse(sys.error("Driver does not exist"))
+  implicit def driver: WebDriver = _driver.getOrElse(sys.error("Driver does not exist"))
```

Comment:
this shouldn't be used outside of this trait, it's unsafe in that it will throw if `_driver` doesn't exist - I only put it in here because things extending `Steps` had the guarantee that  a test was being run so the `Before` and `After` blocks were being called below